python/examples/learn/4-nn-modules-equivariant.py

```diff
@@ -0,0 +1,405 @@
+"""
+.. _learn-tutorial-nn-modules-equivariant:
+
+Equivariance-preserving ``nn`` modules
+======================================
+
+.. py:currentmodule:: metatensor.torch.learn.nn
+
+This example demonstrates the use of convenience modules in metatensor-learn to build
+simple equivariance-preserving multi-layer perceptrons.
+
+.. note::
+
+    Prior to this tutorial, it is recommended to read the tutorial on :ref:`using
+    convenience modules <learn-tutorial-nn-modules-basic>`, as this tutorial builds on
+    the concepts introduced there.
+"""
+
+import torch
+from torch import Tensor
+from torch.nn import Module
```

Comment:
```suggestion
```

I would rather be explicit, especially since we might soon™ introduce a `metatensor.learn.nn.Module` class

Reply:
And same for the other tutorials